README.md

```diff
@@ -0,0 +1,80 @@
+# OpenTelemetry Instrumentation: Spring and SpringBoot
+
+This package streamlines the manual instrumentation process in OpenTelemetry for Spring and SpringBoot. It will enable you to add traces to requests, and database calls with minimal changes to application code. This package will not fully automate your OpenTelemetry instrumentation, instead, it will provide you with better tools to instrument your own code.
+This contribution to OpenTelemetry will follow in the footsteps of the existing Spring integration in [OpenCensus](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/spring/src/main/java/io/opencensus/contrib/spring).
+
+This starter guide contains three tutorials. The first tutorial will walk you through span creation and propagation using the JavaSDK and the HTTP client RestTemplate. 
+
+The second tutorial will build on the first. It will walk you through implementing spring-web handler and interceptor interfaces to create traces without modifying your application code. 
+
+The third tutorial will walk you through the annotations and configurations defined in the opentelemetry-contrib-spring package. This tutorial will equip you with new tools to streamline the step up and configuration of OpenTelemetry on Spring and SpringBoot applications.
+
+To use the tools included in this package include the dependency below in your spring application:
+
+
+#### Maven
+```xml
+ <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-contrib-spring</artifactId>
+    <version>VERSION</version>
+ </dependency>
+```
+
+#### Gradle 
+```gradle
+compile "io.opentelemetry:opentelemetry-contrib-spring:VERSION"
+```
+
+## Features (IN PROGRESS)
```

Comment:
This should be one of the sections you defined.

Reply:
This was originally in section 3 but I moved it up to highlight the proposed features in this package (I think it was originally buried on line 600). Since I am using internal links now, it makes more sense to move it back down. The same thing applies to the dependencies in this readme. They belong to section 3.